index.html

```diff
@@ -1,37 +1,78 @@
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Tic Tac Toe</title>
-</head>
+  <body>
+    <div id="mainWrapper">
+      <!--includes name of game, player status, and timer-->
+      <div id="statusWrapper">
+        <div id="title">
+          <h2>TIC TAC TOE</h2>
+        </div>
+        <div id="timer">0</div>
 
-<body>
-  <h2>Tic Tac Toe</h2>
-  <div class="board">
-    <div id="row-1">
-      <div id="cell-0"></div>
-      <div id="cell-1"></div>
-      <div id="cell-2"></div>
+        <div id="status">Lets get started.....</div>
+      </div>
+      <!--this is the interactive game board where play happens-->
+      <div class="board">
+        <div class="boardRow" id="row-1">
+          <div class="cellContainer" id="cell-0">
+            <button class="cellButton"></button>
+          </div>
+          <div class="cellContainer" id="cell-1">
+            <button class="cellButton"></button>
+          </div>
+          <div class="cellContainer" id="cell-2">
+            <button class="cellButton"></button>
+          </div>
+        </div>
+        <div class="boardRow" id="row-2">
+          <div class="cellContainer" id="cell-3">
+            <button class="cellButton"></button>
+          </div>
+          <div class="cellContainer" id="cell-4">
+            <button class="cellButton"></button>
+          </div>
+          <div class="cellContainer" id="cell-5">
+            <button class="cellButton"></button>
+          </div>
+        </div>
+        <div class="boardRow" id="row-3">
+          <div class="cellContainer" id="cell-6">
+            <button class="cellButton"></button>
+          </div>
+          <div class="cellContainer" id="cell-7">
+            <button class="cellButton"></button>
+          </div>
+          <div class="cellContainer" id="cell-8">
+            <button class="cellButton"></button>
+          </div>
+        </div>
+      </div>
+      <!--button that starts the game. option for two player game, playing computer, and once game ends a replay button-->
+      <div class="play-button">
+        <button id="start" class="start-button" type="button">
+          Two Player Game
+        </button>
+        <button id="compStart" class="start-button" type="button">
+          Play Computer
+        </button>
+        <button id="replay" class="start-button" type="button" disabled>
+          Replay
+        </button>
+      </div>
     </div>
-    <div id="row-2">
-      <div id="cell-3"></div>
-      <div id="cell-4"></div>
-      <div id="cell-5"></div>
-    </div>
-    <div id="row-3">
-      <div id="cell-6"></div>
-      <div id="cell-7"></div>
-      <div id="cell-8"></div>
-    </div>
-  </div>
-
-  <div class="status">
-    <button id="start" type="button">Start Game</button>
-  </div>
-  <script src="main.js"></script>
-</body>
-
+    <script src="main.js"></script>
+  </body>
```

Comment:
First of all, great use of flexbox (and supporting css..) to get a super crisp board layout that is pleasant to look at. My only complaint - the board on my screen is just slightly taller than the height of my window. This adds the "ugly" scroll bar to the right side of the window and cuts off those bottom buttons a bit. Just tightening up the horizontal spacing slightly is a quick/easy fix. Although this complaint is very minor.